sim/druid/claw.go

```diff
@@ -0,0 +1,80 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/classic/sim/core"
+)
+
+func (druid *Druid) registerClawSpell() {
+	flatDamageBonus := map[int32]float64{
+		20: 27,
+		28: 39,
+		38: 57,
+		48: 88,
+		60: 115,
+	}[druid.Level]
+
+	druid.Claw = druid.RegisterSpell(Cat, core.SpellConfig{
+		SpellCode: SpellCode_DruidClaw,
+		ActionID: core.ActionID{SpellID: map[int32]int32{
+			20: 1082,
+			28: 3029,
+			38: 5201,
+			48: 9849,
+			60: 9850,
+		}[druid.Level]},
+		SpellSchool: core.SpellSchoolPhysical,
+		DefenseType: core.DefenseTypeMelee,
+		ProcMask:    core.ProcMaskMeleeMHSpecial,
+		Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagAPL | SpellFlagOmen | SpellFlagBuilder,
+
+		EnergyCost: core.EnergyCostOptions{
+			Cost:   45 - 1*float64(druid.Talents.Ferocity),
+			Refund: 0.8,
+		},
+		Cast: core.CastConfig{
+			DefaultCast: core.Cast{
+				GCD: time.Second,
+			},
+			IgnoreHaste: true,
+		},
+
+		DamageMultiplier: 1,
```

Comment:
Claw benefits from Savage Fury, so I believe this should be

`DamageMultiplier: 1 + 0.1*float64(druid.Talents.SavageFury)`

To match how Rake is implemented.

Reply:
Both should actually use `DamageMultiplierAdditive` for the savage fury contribution

Reply:
I added Savage Fury additive damage modifier to Claw and changed Rake's to additive as well.